src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+class ColorSupplier {
+    private static final Random random = new Random();
+
+    public static Color getRandomColor() {
```

Comment:
The method should be non-static
```suggestion
    public Color getRandomColor() {
```